cfgov/unprocessed/css/pages/home.less

```diff
@@ -17,6 +17,14 @@
 }
 
 .o-well__home {
+    max-width: 1230px;
```

Comment:
This should be set to `@grid_wrapper-width`, I think.

Reply:
Yep, good catch.